CHANGELOG.md

```diff
@@ -7,6 +7,29 @@ the `CHANGELOG.md` for each individual package for any changes relevant for deve
 If you are looking for the Ledger Releasing and Versioning Process then you can find it in
 [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd).
 
+## 8.5
+
+- Prevent `DRep` expiry when there are no Governance Actions.
+- Refreshing of expiry with votes and `UpdateTxCert`
+- Disable ability to submit transactions with votes by Stake Pool Operators and
+  Constitutional Committee members on Governance Actions that they should not
+  be able to vote on.
+- Add ability to supply initial Constitutional Committee and initial version of Contitution.
+- Preserve order of submitted ProposalProcedures and account for their priority.
+- Disallow empty fields in CBOR of Conway TxBody
+- Implement Constitutional Committee expiration, validation and modification
+
```

Comment:
How about [this](https://github.com/input-output-hk/cardano-ledger/commit/16a9090d62bacaf441a4da9726cf2483d006e0ee) commit ?  `Add conway-specific certs to deposit/refunds`

But I don't understand it very well, maybe it's not something that is reflectable in changelog..., please ignore if so